app/components/MachinesPage/MachinesQuoteForm.tsx

```diff
@@ -0,0 +1,129 @@
+        {props?.requestQuoteForm?.formTitle || "Request a Quote"}
+      </Heading>
+      <VStack
+        spacing={6}
+        px={8}
+        py={10}
+        borderRadius="lg"
+        boxShadow="xl"
+        maxW="6xl"
+        mx="auto"
+      >
+        <HStack w="full" spacing={6}>
+          <FormControl isRequired>
+            <Input
+              bg="white"
+              borderColor="white"
+              focusBorderColor="brand.accentGreen"
+              borderRadius="full"
+              border="2px"
+              id="name"
+              placeholder={props?.requestQuoteForm?.field1Placeholder}
+              _placeholder={{ opacity: 1, color: "brand.accentGrey" }}
+              aria-label="Name"
+              _hover={{
+                borderColor: "brand.accentGreen",
+              }}
+              {...register("name", { required: true })}
+            />
+          </FormControl>
+          <FormControl isRequired>
+            <Input
+              bg="white"
+              borderColor="brand.primary"
+              focusBorderColor="brand.accentGreen"
+              borderRadius="full"
+              border="2px"
+              id="email"
+              type="email"
+              placeholder="Email"
+              _placeholder={{ opacity: 1, color: "brand.accentGrey" }}
+              aria-label="Email"
+              _hover={{
+                borderColor: "brand.accentGreen",
+              }}
+              {...register("email", { required: true })}
+            />
+          </FormControl>
+          <FormControl>
+            <Input
+              bg="white"
+              borderColor="brand.primary"
+              focusBorderColor="brand.accentGreen"
+              borderRadius="full"
+              border="2px"
+              id="industry"
+              placeholder={props?.requestQuoteForm?.field2Placeholder}
+              _placeholder={{ opacity: 1, color: "brand.accentGrey" }}
+              aria-label="Industry type"
+              _hover={{
+                borderColor: "brand.accentGreen",
+              }}
+              {...register("industry")}
+            />
+          </FormControl>
+        </HStack>
+        <FormControl>
+          <Textarea
+            bg="white"
+            borderColor="brand.primary"
+            focusBorderColor="brand.accentGreen"
+            borderRadius="3xl"
+            border="2px"
+            id="details"
+            placeholder="Share additional information or questions you have. Our team will personally connect with you about this quote."
+            _placeholder={{ opacity: 1, color: "brand.accentGrey" }}
+            aria-label="Share additional information or questions"
+            _hover={{
+              borderColor: "brand.accentGreen",
+            }}
+            {...register("details")}
+          />
+        </FormControl>
+        <Button px={12} py={6} my={8} variant="mc-red" w="md" type="submit">
```

Comment:
```suggestion
        <Button px={12} py={6} my={8} variant="mc-red" fontSize="2xl" w="md" type="submit">
```
In wireframes this appears larger